components/MW/metgene_metabolite_table/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "metgene_metabolite_table",
+  "version": "1.0.0",
+  "license": "CC-BY-NC-SA-4.0",
+  "author": "",
```

Comment:
Add author

Reply:
Done

Reply:
When done you can just click the "Resolve conversation" button :)